src/inverseRobot.js

```diff
@@ -8,6 +8,23 @@
 
 function inverseRobot(robot) {
   // write code here
+  const invertedRobot = {};
+  const valuesSeen = {};
+
+  for (const key in robot) {
+    if (robot.hasOwnProperty(key)) {
```

Comment:
The 'hasOwnProperty' check is unnecessary in this for..in loop as we're not dealing with inherited properties in this context. It can be safely removed to cleanup the code.